politeiawww/memorycache.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"sync"
+	"time"
+)
+
+// Item is a cached reference
+type Item struct {
+	Content    []byte
+	Expiration int64
+}
+
+//Storage mechanism for caching strings in memory
```

Comment:
```suggestion
// Storage mechanism for caching strings in memory
```

Reply:
There are a couple other instances of this that should be cleaned up.

Reply:
Also, make sure that all the function declaration comments are full senteneces and use proper punctuation.  This one is missing a period.  The reason is because exported function declarations comments get pulling into the go docs so should be proper sentences.  Unexported function comments should use the same style as exported function comments to keep things consistent.

Reply:
ok now ?